### ISSUE TITLE
Introduce facette for inCollection and list inCollection in detail page

### ISSUE THREAD

**Issue by TobiasNx** [hbz/lobid-resources#2109]
To better filter resources for their collection in the GUI, it would good to have a facette for inCollection.

Also add the inCollection values to the detail page of a resource.

**Comment by acka47** [comment]
Assigning @fsteeg to add this during summer.

**Comment by fsteeg** [comment]
Deployed to test: https://test.lobid.org/resources/search

**Comment by TobiasNx** [comment]
Thanks for the explanation we have some URIs from lobid-resources and some self defined in there.
Also the alternateName in lobid organisations can exists more than one time, see for the example above:
https://lobid.org/organisations/ZDB-1-AIS#!

```
 "alternateName": [
    "NL FID-Lizenz Almanhal",
    "Nationallizenz Almanhal",
    "Almanhal Islamic Studies E-Book Collection [FID-Lizenz] (früherer Name)"
  ],

**Comment by fsteeg** [comment]
> It would make more sense if only the first would be picked: "NL FID-Lizenz Almanhal"

Yes, it seems using the last was a workaround (see https://github.com/hbz/lobid-resources/commit/6cc1ce46318f2ef467c21d3c2cf2b4b0d2d5ffb6), switched to first on test:

https://test.lobid.org/resources/search?collection=http%3A%2F%2Flobid.org%2Forganisations%2FZDB-1-AIS%23%21&aggregations=inCollection.id

> [...] perhaps it makes sense to add a lookup for lobid-resources

Deployed to test: https://test.lobid.org/resources/search

> Otherwise @acka47 should decide.

Assigning @acka47 for review.

**Comment by acka47** [comment]
This looks quite good but we need to tweak it a bit more, I think:

- In the detail view, we should use a German field name, I think "Kollektionen" should be ok as most resources are part of at least two collections, aren't they?
- In the filter, we should have human-readable names at least for the biggest collections:
   - https://nrw.digibib.net/search/hbzvk/ -> "hbz-Verbundkatalog"
   -   https://lobid.org/collections#NLZ  -> "Nationallizenzen"
   - "hbz" should rather read "Alma-NZ" or "Alma-Netzwerkzone"
   - Also, [https://oersi.org/ -> "OERSI" would  be nice.

**Comment by fsteeg** [comment]
>  I think "Kollektionen" should be ok [...] we should have human-readable names at least for the biggest collections

I've renamed the facet and field name to "Kollektionen" and added 3 custom labels for the non-Organisations / non-Resources URLs. Deployed to  test: https://test.lobid.org/resources/search

> "hbz" should rather read "Alma-NZ" or "Alma-Netzwerkzone"

I think that should rather be fixed in https://lobid.org/organisations/DE-655 (`alternateName`).

**Comment by acka47** [comment]
Thanks,  this is now ok for me. +1
